tests/test_declarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -25,13 +21,11 @@ it("should create the app", async(() => {
   // code
 }));
 
-it(\`should have as title 'app'\`, async(() => {
-  // code
-}));
-
-it("should render title in a h1 tag", async(() => {
-  // code
-}));
+it("does something really long and complicated so I have to write a very long name for the test", async(
+  () => {
```

Comment:
I think this function expression should hug the `async` call expression in order to be consistent with the other inlined test patterns.

Reply:
Oh that's probably what you meant by it only fixing part of the problem